java-rmi/src/com/baeldung/rmi/MessengerServiceImpl.java

```diff
@@ -0,0 +1,49 @@
+package com.baeldung.rmi;
+
+import java.rmi.RemoteException;
+import java.rmi.registry.LocateRegistry;
+import java.rmi.registry.Registry;
+import java.rmi.server.UnicastRemoteObject;
+
+public class MessengerServiceImpl implements MessengerService {
+
+	@Override
+	public String sendMessage(String clientMessage) {
```

Comment:
The code and the blog are mismatched here. Please change this method to "send".

Reply:
Will do this.

Reply:
I think we should change the article from send() to sendMessage() for this.

Reply:
Done.